locale/fr/about/governance.md

```diff
@@ -0,0 +1,147 @@
+maintenu par le TSC et un groupe de Collaborateurs additionnels
+qui sont ajoutés par le TSC de manière continue.
+
+Les personnes proposant des contributions significatives sont faits
+Collaborateurs et se voient accorder les droits d'écriture sur le projet.
+Ces personnes sont identifiées par le TSC et leur ajout aux Collaborateurs
+est discuté lors des réunions hebdomadaires du TSC
+
+_Note:_ Si vous faites des contributions significatives et n'avez pas encore
+obtenu les droits de commit, ouvrez un ticket ou contactez un membre du TSC
+directement et votre demande sera examinée lors de la prochaine réunion.
+
+Les modifications de contenu sur le dépôt nodejs/node sont validées de
+manière collaborative. N'importe qui possédant un compte GitHub peut
+proposer une modification par pull request et elle sera examinée par les
+Collaborateurs du projet. Toutes les pull request doivent être relues et acceptées
+par un Collaborateur ayant une expertise suffisante et qui soit capable
+d'assumer la responsabilité du changement effectué. Dans le cas d'une pull
+request proposée par un Collaborateur, un autre Collaborateur est requis
+pour valider la modification. Un consensus doit être recherché si
+d'autres Collaborateurs participent et qu'un désaccord survient sur
+une modification. Voyez _Processus de Recherche de Consensus_ plus
+bas pour plus de détails sur le modèle de consensus utilisé.
+
+Les Collaborateurs peuvent choisir de faire remonter au TSC pour
+discussion des modifications significatives ou sujettes à controverse,
+ou des modifications n'ayant pas obtenu de consensus, en leur assignant
+l'étiquette ***tsc-agenda*** sur une pull request ou un ticket. Le
+TSC servira alors d'arbitre final lorsque requis.
+
+Pour la liste des Collaborateurs actuels, voir le projet 
+[README.md](https://github.com/nodejs/node/blob/master/README.md#current-project-team-members).
+
+Un guide des Collaborateurs est proposé sur 
+[COLLABORATOR_GUIDE.md](https://github.com/nodejs/node/blob/master/COLLABORATOR_GUIDE.md).
+
+## Siéger au TSC
+
+Les sièges au TSC ne sont pas limités dans le temps. Le TSC n'a pas
+de taille fixe. Cependant, la cible est entre 6 et 12 membres, afin
+d'assurer une couverture suffisante des différents domaines d'expertise,
+en conservant la possibilité de prendre des décisions de manière
+efficace.
+
+Il n'y a aucun prérequis ou qualifications nécessaires au-delà de
+ces règles pour siéger au TSC.
+
+Le TSC peut ajouter des membres additionnels au TSC par une motion
+standard du TSC.
+
+Un membre du TSC peut être retiré du TSC par démission volontaire, ou
+par une motion standard du TSC.
+
+Les changements de modalités de participation au TSC devront être mises
+au programme et peuvent être ajoutés comme n'importe quel autre
+objet de programme (voir "Réunions du TSC" plus bas).
+
+Pas plus d'1/3 des membres du TSC ne pourront être affiliés au même
+employeur. Si le retrait ou la démission d'un membre du TSC, ou le 
```

Comment:
In french,  [1/3 is singular and should be written in full letters](http://www.aidenet.eu/grammaire04k.htm), not sure about english.

> Pas plus d'un tiers des membres du TSC ne pourra être affilié au même employeur.